docs/source/evaluation_scripts.md

```diff
@@ -129,3 +129,205 @@ Here are the steps to configure remote evaluation:
+                self.server = server
+
+            def get_action_space(self, request, context):
+                message = pack_for_grpc(env.get_action_space())
+                return evaluation_pb2.Package(SerializedEntity=message)
+
+            def act_on_environment(self, request, context):
+                global EVALUATION_COMPLETED
+                if not env.feedback or not env.feedback[2]:
+                    action = unpack_for_grpc(request.SerializedEntity)
+                    env.next_score()
+                    env.feedback = env.env.step(action)
+                if env.feedback[2]:
+                    if not LOCAL_EVALUATION:
+                        update_submission_result(
+                            env, self.challenge_pk, self.phase_pk, self.submission_pk
+                        )
+                    else:
+                        print("Final Score: {0}".format(env.score))
+                        print("Stopping Evaluation!")
+                        EVALUATION_COMPLETED = True
+                return evaluation_pb2.Package(
+                    SerializedEntity=pack_for_grpc(
+                        {"feedback": env.feedback, "current_score": env.score,}
+                    )
+                )
+        ```
+
+        You can modify the relevant parts of the environment service in order to make it work for your case. For example, `feedback` can totally depend on the results obtained from the `env.env` after performing an action, which in-turn depends on the sensors expected from the agent. In our example, `feedback[2]` is the stop signal sensor. The `unpack_for_grpc` and `pack_for_grpc` are methods to receive and send messages with the agent.
+
+        This is a high-level description of the class and the implementations may vary on a case-by-case basis.
+    3. _Edit the requirements file_: Change the [requirements file](https://github.com/Cloud-CV/EvalAI-Starters/blob/master/code_upload_challenge_evaluation/requirements/environment.txt) according to the packages required by your environment.
+
+    4. _Edit environment Dockerfile_: You may choose to modify the [Dockerfile](https://github.com/Cloud-CV/EvalAI-Starters/blob/master/code_upload_challenge_evaluation/docker/environment/Dockerfile) which will host the Environment service on a GRPC server.
+
+    5. _Edit the docker environment variables_: Fill in the following information in the [`docker.env`](https://github.com/Cloud-CV/EvalAI-Starters/blob/master/code_upload_challenge_evaluation/docker/environment/docker.env) file:
+
+        ```env
+        AUTH_TOKEN=<Add your EvalAI Auth Token here>
+        EVALAI_API_SERVER=<https://eval.ai>
+        LOCAL_EVALUATION = True
+        QUEUE_NAME=<Go to the challenge manage tab to get challenge queue name.>
+        ```
+
+    6. _Create the docker image and host it_: Create an environment docker image using the above steps using `docker build` command and host it.
```

Comment:
Just say - build the environment docker by running the following command `docker build -f <file_name>`

basically show them the example. What does "host it" mean here?

Reply:
Tell them to build docker and upload to the ECR and share a sample command to upload to ECR

Reply:
Host it basically meant either over ECR or DockerHub or somewhere.

Reply:
Added sample steps.